htdocs/core/class/conf.class.php

```diff
@@ -161,6 +174,58 @@ public function __construct()
 	 */
 	public function setValues($db)
 	{
+		// Unset all old modules values
+		if (!empty($this->modules)) {
```

Comment:
The whole $this->modules is completely erased few lines after with
$this->modules = array();
So i think we can remove this portion of code, doesn't it ?